src/main/java/by/itechart/api/controller/UserController.java

```diff
@@ -26,22 +29,26 @@
      *
      * @return current user
      */
+    @ApiOperation(value = "Get current user", notes = "This method will return current user")
     @GetMapping("/current")
     public ResponseEntity<UserDTO> getCurrentUser() {
         return new ResponseEntity<>(HttpStatus.FOUND);
     }
 
+    @ApiOperation(value = "Update user", notes = "This method is used to update user depending on user's id")
     @PatchMapping("{id}")
     public ResponseEntity<UserDTO> updateUser(@PathVariable Long id, @RequestBody UserDTO user) {
```

Comment:
You can add `@ApiParam` annotation with parameters description here and in other places